skpro/datatypes/_check.py

```diff
@@ -263,7 +258,11 @@ def check_raise(obj, mtype: str, scitype: str = None, var_name: str = "input"):
         raise TypeError(msg)
 
 
-def mtype(obj, as_scitype=None, exclude_mtypes=AMBIGUOUS_MTYPES):
+def mtype(
+    obj,
+    as_scitype,
```

Comment:
why is the default being removed? This is a breaking change, so should be reverted. If inconsistent with `sktime`, consider adding the default in `sktime`.

Reply:
I accidently removed the `None` when i got rid of the type annotations